language/entity-extraction/src/test/java/com/google/cloud/language/automl/entity/extraction/samples/ModelIT.java

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.language.automl.entity.extraction.samples;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.PrintStream;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/**
+ * Tests for AutoML Natural Language Entity Extraction Model operations.
+ */
+@RunWith(JUnit4.class)
+public class ModelIT {
+
+  private static final String PROJECT_ID = System.getenv().get("GOOGLE_CLOUD_PROJECT");
+  private static final String COMPUTE_REGION = "us-central1";
+  private static final String FILTER = "textExtractionModelMetadata:*";
+  private static final String MODEL_ID = "TEN1974951581904273408";
```

Comment:
@beccasaurus, I'm not sure how AutoML samples have handled this in the past, using a special model ID to test, since waiting to train can take hours. Just makes our tests likely to fail for other people. 

I feel like this is a good solution, kinda just one of those tough areas to come up with a solution for, thoughts?

Reply:
We have used pre-built models for our other AutoML tests, but yes - they will fail for other people.

Reply:
Ah ok, cool! Then works for me!

Wasn't sure if we had done that before and seems like the best option.